README

```diff
@@ -1133,7 +1133,8 @@ NETWORKING SUPPORT / OPTIONS
   Specify the directory where the OpenFabrics Interfaces libfabric
   library and header files are located.  This option is generally only
   necessary if the libfabric headers and libraries are not in default
-  compiler/linker search paths.
```

Comment:
the bot:notacherrypick should be in the PR body, but *not* the commit body.  Please update your commit message.

Reply:
Updated.